.github/workflows/main.yml

```diff
@@ -723,6 +729,18 @@ jobs:
         rustup target add ${{ matrix.target }}
       if: matrix.target != ''
 
+    - run: rustup target add aarch64-linux-android
+    - run: rustup target add x86_64-linux-android
+    - name: Setup Android SDK
+      uses: android-actions/setup-android@v2
+    - uses: actions/cache@v3
+      with:
+        path: ${{ runner.tool_cache }}/cargo-ndk
+        key: cargo-ndk-bin-${{ env.CARGO_NDK_VERSION }}
+    - run: echo "${{ runner.tool_cache }}/cargo-ndk/bin" >> $GITHUB_PATH
+    - run: cargo install --root ${{ runner.tool_cache }}/cargo-ndk --version ${{ env.CARGO_NDK_VERSION }} cargo-ndk
+    - run: cargo ndk -t arm64-v8a -t x86_64 check -p wasmtime
```

Comment:
I think this can be omitted since this is a build step rather than a "check" step